picard/acoustid/manager.py

```diff
@@ -37,6 +37,8 @@ def __init__(self, fingerprint, duration, orig_recordingid=None, recordingid=Non
 
 class AcoustIDManager(QtCore.QObject):
 
+    BATCH_SUBMIT_COUNT = 240
```

Comment:
Please explain (in code comment) why the value of 240 was chosen (is it recommended somewhere or just a "good" value?)